signac/contrib/project.py

```diff
@@ -142,7 +142,12 @@ def __init__(self, config=None):
         self._config = _ProjectConfig(config)
 
         # Ensure that the project id is configured.
-        self.get_id()
+        id_ = self.get_id()
+        if id_ is None:
+            raise LookupError(
+                "Unable to determine project id ."
```

Comment:
```suggestion
                "Unable to determine project id."
```

Reply:
Actually you need a space after the period so this concatenates properly with the following line.